src/GameLogic.cpp

```diff
@@ -200,11 +209,25 @@ void GameLogic::increaseScore(int level, int increase) {
     }
 }
 
-int GameLogic::getScore(int level) {
+unsigned int GameLogic::getScore(int level) {
+    if (level < 0 || level > 9) {
+        return 0;
+    }
     return scores[level];
 }
 
-bool GameLogic::setScore(int level, int score) {
+std::vector<unsigned int> GameLogic::getScoresVector() {
```

Comment:
I don't think we should have a method that just passes the entire vector. You can get everything in the vector by calling getScore, so I think this violates privacy a bit, if you want to do this then just make scoresVector public